Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -39,16 +41,53 @@ AddNewCardViewModelOutputs {
 
   public init() {
     let cardholderName = self.cardholderNameChangedProperty.signal.skipNil()
-    let creditCardDetails = self.creditCardChangedProperty.signal.skipNil()
+    let creditCardDetails = self.creditCardChangedProperty
+      .signal
+      .skipNil()
+      .filterMap { args -> (String, UInt, UInt, String)? in
+        let (cardNumberArg, expMonthArg, expYearArg, cvcArg) = args
+
+        guard let expMonth = expMonthArg,
```

Comment:
Looks like this whole `guard` would fit on one line.

If this is preferred way is there a reason why the 2nd and 3rd lines were indented manually instead of using the default spacing? Just curious.

This
```
guard
          let expMonth = expMonthArg,
          let expYear = expYearArg,
          let cvc = cvcArg else {
            return nil
        }
```
instead of this
```
guard let expMonth = expMonthArg,
           let expYear = expYearArg,  // manual indentation
           let cvc = cvcArg else {       // manual indentation
          return nil
        }
```

Reply:
I just find it to be more readable with long `guard` statements with multiple variables being unwrapped if each variable is on it's own line. It's personal preference but we can discuss what the team would prefer to go with for consistency.